### PR TITLE
Add a section in the configuration about flushing.

### PR DESCRIPTION
**Pull request by ametaireau**
I'm actually not sure if it is the best place to do that. Another location I've been looking for this information was the endpoints, so it might be better to add it there.

In any case, we need to document this :-)

Thoughts?

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -220,6 +220,23 @@ To use a different ini file, the ``KINTO_INI`` environment variable
 should be present with a path to it.
 
 
+Activating the flush endpoint
+=============================
+
+When using Kinto in development mode, it might be helpful to have a way to
+flush all the data that is currently stored in the database.
```
Comment (Natim) [inline docs/configuration.rst]: that are currently stored (data is always a plural in english)

**Comment by Natim** [comment]
LGTM with nit. I like it to be in the configuration section.

File: docs/configuration.rst (hunk 1 above)
Comment (ametaireau) [inline docs/configuration.rst]: ah yes, I'm always doing this error :x

**Comment by ametaireau** [comment]
@leplatrem thoughts?

File: docs/configuration.rst (hunk 1 above)
Comment (ametaireau) [inline docs/configuration.rst]: I removed the "that is" ;)

File: docs/configuration.rst [hunk 2]
```diff
@@ -220,6 +220,23 @@ To use a different ini file, the ``KINTO_INI`` environment variable
...
+
+There is a way to enable this behaviour (it is deactivated by default for
+obvious security reasons). In your `.ini` file:
+
+.. code-block :: ini
+
+    kinto.flush_endpoint_enabled = true
+
+Then, you can issue a `POST` request to the `/__flush__` endpoint to flush all
+the data.
```
Comment (leplatrem) [inline docs/configuration.rst]: -you

Then, a POST request [...] flushes all the data

**Comment by leplatrem** [comment]
r+ with «you nit»

**Pull request merged**